templates/publisher/listing.html

```diff
@@ -183,7 +183,7 @@ <h2 class="p-heading--4">
       <div class="col-8">
         <div class="p-form__control">
           <input class="p-form-validation__input" type="text" name="video_urls" value="" id="video">
-          <p class="p-form-help-text">
+          <p class="p-form-help3-text">
```

Comment:
shouldn't this be `p-form-help-text`? I can't find `p-form-help3-text` in the codebase?

Reply:
hahaha. I'll fix it. thanks